calblueprint/girls-write-now
# [retool] Add way to set description of featured stories

[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR
* Table created in retool to update description and header per story

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

## Relevant Links
N/A

### Notion Sprint Task
https://www.notion.so/calblueprint/retool-Add-way-to-set-description-of-featured-stories-b8d3a39aa9304d7097f0fcc45ed81dc8

[//]: # 'Add the relevant Notion link(s) here'

### Online sources
N/A

[//]: # 'Optional - copy links to any tutorial or documentation that was useful to you when working on this PR'

### Related PRs
N/A

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

## How to review
* Run app and check the header/description
* Change it in retool to something else and it should update in the sup abase as well

[//]: # 'The order in which to review files and what to expect when testing locally'

## Next steps
* Unsure if the new table should be connected to the other featured stories table and if they should have the same listings. Can thus change retool tables to be the same and to change the sup abase. If there should only be one entry, I can also just leave on entry that is not attached to any story. Was unsure if it should overall be in sync with each story or if we just want one overall entry.

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

## Tests Performed, Edge Cases
N/A

[//]: # 'Hopefully we will add a testing suite/CI soon, but until then note down the steps you took to test locally'

### Screenshots
![Screenshot 2024-04-21 at 6 12 55 PM](https://github.com/calblueprint/girls-write-now/assets/69034384/9fbaa489-d388-4b9e-8643-440cfb216dec)
![Screenshot 2024-04-21 at 6 13 10 PM](https://github.com/calblueprint/girls-write-now/assets/69034384/6629febe-668d-4131-8641-1f2b294c5a03)


[//]: #
![IMG_2318](https://github.com/calblueprint/girls-write-now/assets/69034384/02d2cea6-a993-4972-b2be-fb5138b85b60)
 "Add screenshots of expected behavior - GIFs if you're feeling fancy!"

CC: @adityapawar1

[//]: # 'This tags in Adi as a default. Feel free to change, or add on anyone who you should be in on the conversation.'
